modules/travel_pay/app/services/travel_pay/token_client.rb

```diff
@@ -116,19 +87,21 @@ def build_sts_assertion(user)
       }
     end
 
-    def build_host_baseurl(config)
-      env = Settings.vsp_environment
-      host = Settings.hostname
+    private
 
-      if env == 'localhost'
-        if config[:ip_form]
-          return 'http://127.0.0.1:3000'
-        else
-          return 'http://localhost:3000'
-        end
+    def claim_headers
+      if Settings.vsp_environment == 'production'
+        {
+          'Content-Type' => 'application/json',
+          'Ocp-Apim-Subscription-Key-E' => Settings.travel_pay.subscription_key_e,
+          'Ocp-Apim-Subscription-Key-S' => Settings.travel_pay.subscription_key_s
+        }
+      else
+        {
+          'Content-Type' => 'application/json',
+          'Ocp-Apim-Subscription-Key' => Settings.travel_pay.subscription_key
+        }
```

Comment:
See comment above about moving this and the `connection` method to a base `TravelPay::Client` class